LocalSettings.archlinux.org.php

```diff
@@ -0,0 +1,378 @@
+##
+
+# DeveloperWiki
+define("NS_DEVELOPERWIKI", 3000);
+define("NS_DEVELOPERWIKI_TALK", 3001);
+$wgExtraNamespaces[NS_DEVELOPERWIKI] = "DeveloperWiki";
+$wgExtraNamespaces[NS_DEVELOPERWIKI_TALK] = "DeveloperWiki_talk";
+$wgNamespacesWithSubpages[NS_DEVELOPERWIKI] = true;
+$wgNamespacesWithSubpages[NS_DEVELOPERWIKI_TALK] = true;
+$wgContentNamespaces[] = NS_DEVELOPERWIKI;
+$wgNamespacesToBeSearchedDefault[NS_DEVELOPERWIKI] = true;
+$wgNamespaceProtection[NS_DEVELOPERWIKI] = array( 'editprotected2' );
+
+
+##
+## Additional extensions
+##
+
+wfLoadExtension( 'Nuke' );
+
+# AbuseFilter extension
+wfLoadExtension( 'AbuseFilter' );
+$wgGroupPermissions['sysop']['abusefilter-modify'] = true;
+$wgGroupPermissions['*']['abusefilter-log-detail'] = true;
+$wgGroupPermissions['*']['abusefilter-view'] = true;
+$wgGroupPermissions['*']['abusefilter-log'] = true;
+$wgGroupPermissions['sysop']['abusefilter-private'] = true;
+$wgGroupPermissions['sysop']['abusefilter-modify-restricted'] = true;
+$wgGroupPermissions['sysop']['abusefilter-revert'] = true;
+
+# filter groups
+$wgAbuseFilterValidGroups = array( 'default', 'proofed' );
+$wgAbuseFilterEmergencyDisableThreshold = array(
+    'default' => 0.5,
+    'proofed' => 1.0,
+);
+$wgAbuseFilterEmergencyDisableCount = array(
+    'default' => 10,
+    'proofed' => 65535,
+);
+
+# CheckUser extension
+wfLoadExtension( 'CheckUser' );
+$wgGroupPermissions['sysop']['checkuser'] = true;
+$wgGroupPermissions['sysop']['checkuser-log'] = true;
+
+# ParserFunctions extension
+wfLoadExtension( 'ParserFunctions' );
+
+# Interwiki extension
+wfLoadExtension( 'Interwiki' );
+$wgGroupPermissions['sysop']['interwiki'] = true;
+
+# TitleKey extension
+wfLoadExtension( 'TitleKey' );
+
+# BounceHandler extension
+wfLoadExtension( 'BounceHandler' );
+$wgVERPprefix = "wiki-bounce";
+$wgVERPdomainPart = "archlinux.org";
+
+# CAPTCHA
+# The dynamic $wgCaptchaQuestions is defined in the infrastructure repository
+wfLoadExtensions([ 'ConfirmEdit', 'ConfirmEdit/QuestyCaptcha' ]);
+$wgCaptchaTriggers['edit'] = false;
+$wgCaptchaTriggers['create'] = false;
+$wgCaptchaTriggers['addurl'] = false;
+$wgCaptchaTriggers['createaccount'] = true;
+$wgCaptchaTriggers['badlogin'] = true;
+
+# Restrict expensive actions to logged in users
+wfLoadExtension( 'Lockdown' );
+$wgSpecialPageLockdown['Recentchanges'] = [ 'user' ];
+$wgSpecialPageLockdown['Newpages'] = [ 'user' ];
+$wgSpecialPageLockdown['Recentchangeslinked'] = [ 'user' ];
+$wgSpecialPageLockdown['Log'] = [ 'user' ];
+$wgSpecialPageLockdown['Diff'] = [ 'user' ];
+$wgActionLockdown['history'] = ['user'];
+
+##
+## Temporary settings for maintenance
+##
+
+# temporary disable registration; reverted 2016-6-26 bluewind
+#$wgGroupPermissions['*']['createaccount'] = false;
+
+# $wgShowSQLErrors = true;
+# $wgReadOnly = 'Database migration in progress. We`ll be back in a few minutes.';
+# made the wiki have a dark mode
+#wfLoadExtension( 'DarkMode' );
+$wgDarkModeTogglePosition
+#sidebar
+
```

Comment:
The `LocalSettings.archlinux.org.php` file wasn't [removed](https://github.com/archlinux/archwiki/commit/058c6cbd5b0eb24934afab5b34147a060920922b) without a reason. Even if you add it back, nothing will use it.

ArchWiki's `LocalSettings.php` is in the infrastructure repo: https://gitlab.archlinux.org/archlinux/infrastructure/-/blob/master/roles/archwiki/templates/LocalSettings.php.j2.